child_compassion/models/project_reservation.py

```diff
@@ -0,0 +1,42 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class ProjectReservation(models.Model):
+
+    _name = 'compassion.reservation'
+    _description = 'Project Reservation'
+
+    beneficiary_global_id = None
```

Comment:
Ah yes so you can completely remove this line. However you will need to add a field for `Reservation_ID`, because you will receive it in the answer and you want to store it.